partner_street_number/models/res_partner.py

```diff
@@ -41,26 +42,26 @@ def _write_street(self):
         us with a way of easily restoring the data when this module is
         installed on a database that already contains addresses).
         """
-        street_name = self.street and self.street.strip() or False
-        street_number = False
-        if self.street:
-            match = re.search(r'(.+)\s+(\d.*)', self.street.strip())
-            if match and len(match.group(2)) < 6:
-                street_name = match.group(1)
-                street_number = match.group(2)
         for partner in self:
-            self.street_name = street_name
-            self.street_number = street_number
+            street_name = partner.street and partner.street.strip() or False
```

Comment:
Can you use inline if:

`partner.street.strip() if partner.street else False`

?